applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -430,17 +560,18 @@ static void uart_callback(struct uart_event *evt, void *user_data)
 		LOG_INF("TX_ABORTED");
 		break;
 	case UART_RX_RDY:
-		uart_rx_handler(evt->data.rx.buf[0]);
+		for (int i = pos; i < (pos + evt->data.rx.len); i++) {
+			uart_rx_handler(evt->data.rx.buf[i]);
+		}
+		pos += evt->data.rx.len;
 		break;
 	case UART_RX_BUF_REQUEST:
-		err = uart_rx_buf_rsp(uart_dev, &uart_rx_buf[buf_num], 1);
-		if (err) {
```

Comment:
Please keep this logging, with which we could identify Serial interface issue

Reply:
Keep the log as:

		err = uart_rx_buf_rsp(uart_dev, next_buf,
					sizeof(uart_rx_buf[0]));
		if (err) {
			LOG_WRN("UART RX buf rsp: %d", err);
		}